plugins/kubernetes/app/models/kubernetes/deploy_executor.rb

```diff
@@ -142,6 +142,20 @@ def fetch_pods
       end
     end
 
+    def show_pods_logs
+      log_end_time = 0.seconds.from_now
+      pods = fetch_pods
+      pods.each do |pod|
+        if pod.completed? && pod.annotations[:'samson/show_logs_on_success']
```

Comment:
better check for
`pod.annotations[:'samson/show_logs_on_success'] == 'true'`

Reply:
... since showing logs has a timeout, there is no real reason to not allow it for running pods if the user requested it

Reply:
Ohh, it makes sense! I removed that check! :)